HackIllinois/Misc/Constants.swift

```diff
@@ -11,21 +11,65 @@
 //
 
 import Foundation
+import CoreData
+import HIAPI
 
 // MARK: - Constants
-struct HIConstants {
+class HIConstants {
+
+    static var shared = HIConstants()
+
+    func updateTimes() {
+
+        // Update the times of event
+        TimeService.getTimes()
+            .onCompletion { result in
+                do {
+                    let (times, _) = try result.get()
+                    let timesDict = times.data
+                    guard let eventStartTime = timesDict["eventStart"],
```

Comment:
Parsing would be better if you had a Time struct that was decodeable and just return that directly, Swift will parse it into the struct for you.

Reply:
Done